.pre-commit-config.yaml

```diff
@@ -1,27 +1,36 @@
+---
 default_stages: [commit]
 default_language_version:
   python: python3.7
 repos:
-- repo: local
-  hooks:
-  - id: isort
-    stages: [commit]
-    name: isort
-    entry: bash -c 'pipenv run isort --check .'
-    language: system
-    types: [python]
-  - id: flake8
-    stages: [commit]
-    name: flake8
-    entry: bash -c 'pipenv run flake8 .'
-    language: system
-  - id: mypy
-    stages: [commit]
-    name: mypy
-    entry: bash -c 'pipenv run mypy src/build_workflow tests/tests_build_workflow src/checkout_workflow tests/tests_checkout_workflow src/run_assemble.py tests/test_run_assemble.py src/assemble_workflow tests/tests_assemble_workflow src/manifests tests/tests_manifests'
-    language: system
-  - id: pytest
-    stages: [commit]
-    name: pytest
-    entry: bash -c 'pipenv run pytest'
-    language: system
+  - repo: local
+    hooks:
+      - id: isort
+        stages: [commit]
+        name: isort
+        entry: bash -c 'pipenv run isort --check .'
+        language: system
+        types: [python]
+      - id: flake8
+        stages: [commit]
+        name: flake8
+        entry: bash -c 'pipenv run flake8 .'
+        language: system
+      - id: mypy
+        stages: [commit]
+        name: mypy
+        entry: bash -c 'pipenv run mypy src/build_workflow tests/tests_build_workflow
```

Comment:
Looks like `line-length` is disabled in yamllint, which is my preference.  However, it seems like ymlfix was applied with this setting enabled.  Can we synchronize the rules they are applied between them?

Reply:
Unfortunately yaml-fix doesn't support line length configuration (https://github.com/lyz-code/yamlfix/issues/117) currently.

I manually undid the 2-3 instances where the fixer CR/LFed those lines.

Reply:
Maybe we enable the line-length rule so we are consistent, WDYT?

Reply:
Nan, we actually have a similar problem in Python when we recommend using `black` that auto-formats differently, but it hasn't been a concern.